README.md

```diff
@@ -54,11 +54,15 @@
 ---
 
 ## Stats
-[![trophy](https://github-profile-trophy.vercel.app/?username=AswinBarath&margin-w=15)](https://github.com/ryo-ma/github-profile-trophy)
+[![trophy](https://github-profile-trophy.vercel.app/?username=AswinBarath&column=3&margin-w=15&margin-h=15&theme=onedark)](https://github.com/ryo-ma/github-profile-trophy)
+
+[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=AswinBarath&layout=compact)](https://github.com/anuraghazra/github-readme-stats)
 
 ---
```

Comment:
Alright, changing it to a two-column display is good and the dark mode looks better

Reply:
Thanks